scripts/pre_commit_linter.py

```diff
@@ -1312,6 +1334,82 @@ def _check_js_component_name_and_count(self):
             print ''
             return summary_messages
 
+    def _check_mandatory_patterns(self):
+        """This function checks that all JS files contain the mandatory
+        patternss.
+        """
+        if self.verbose_mode_enabled:
+            print 'Starting mandatory patterns check'
+            print '----------------------------------------'
+
+        summary_messages = []
+        failed = False
+        all_files_to_check = [
+            filepath for filepath in self.all_filepaths if not any(
+                fnmatch.fnmatch(filepath, pattern) for pattern in
+                EXCLUDED_PATHS + GENERATED_FILE_PATHS +
+                CONFIG_FILE_PATHS) and not filepath.endswith(
+                    '__init__.py')]
+
+        with _redirect_stdout(_TARGET_STDOUT):
+            for filepath in all_files_to_check:
+                file_content = FileCache.readlines(filepath)
+                # This boolean list keeps track of the regex matches
+                # found in the file.
+                pattern_found_list = []
+                for index, regexp_to_check in enumerate(
+                        MANDATORY_PATTERNS_REGEXP):
+                    if any([filepath.endswith(
+                            allowed_type) for allowed_type in regexp_to_check[
+                                'included_types']]):
+                        pattern_found_list.append(False)
+                        for line in file_content:
+                            if re.search(regexp_to_check['regexp'], line):
+                                pattern_found_list[index] = True
+                                break
+                if not all(pattern_found_list):
+                    failed = True
+                    for index, pattern_found_bool in enumerate(
```

Comment:
I think you can drop the `bool` from `pattern_found_bool`.

Reply:
Here and below.

Reply:
Done!